docs/spec/resources.md

```diff
@@ -82,11 +82,11 @@ resource otherResource 'Microsoft.Example/examples@2020-06-01' = {
 
 **Note:** Conditions are not yet implemented ([#186](https://github.com/Azure/bicep/issues/186)).
 
-Resources may be deployed if and only if a specified condition evaluated to `true`. Otherwise, resource deployment will be skipped. This is accomplished by adding a `when` keyword and a boolean expression to the resource declaration. The template compiled from the below example will deploy the DNS zone if the `deployZone` parameter evaluates to `true`:
+Resources may be deployed if and only if a specified condition evaluated to `true`. Otherwise, resource deployment will be skipped. This is accomplished by adding a `if` keyword and a boolean expression to the resource declaration. The template compiled from the below example will deploy the DNS zone if the `deployZone` parameter evaluates to `true`:
 ```
 param deployZone bool
 
-resource dnsZone 'Microsoft.Network/dnszones@2018-05-01' when (deployZone) = {
+resource dnsZone 'Microsoft.Network/dnszones@2018-05-01' = if (deployZone) {
```

Comment:
We never clarified it in the previous version of the spec, but are we making parentheses required?

Reply:
Good question...I don't think they are needed. We don't uses parentheses for list comprehensions, so it might be better to keep them consistent. What do you think?

Reply:
Personally I vote to require them - just because I feel like it'll seem more familiar to anyone used to C-style languages. We could technically relax this in the future if there's a good case for it. Going from non-required -> required would be a breaking change.

Reply:
> Going from non-required -> required would be a breaking change.

Good point. Let's keep the parentheses then.